pig/src/main/java/org/apache/iceberg/pig/SchemaUtil.java

```diff
@@ -73,6 +60,16 @@ private static ResourceFieldSchema convert(Type type) throws IOException {
     return result;
   }
 
+  private static ResourceFieldSchema [] convertFields(List<Types.NestedField> fields) throws IOException {
```

Comment:
seems like a non functional change? [method moved]

Reply:
Overloaded methods need to be grouped together.